content/_data/logo/smurf.yml

```diff
@@ -0,0 +1,9 @@
+---
+name: 'Smurf'
+url: '/images/logos/smurf/smurf.png'
+url_256: '/images/logos/smurf/256.png'
+vector:
+credit: 'Kseniia Nenasheva'
+credit_url: 'https://twitter.com/ks_nenasheva'
+source: 'Belgium JAM'
```

Comment:
I'll remove this for now. Under U.S. law, the artwork would clearly fall under fair-use, although we wouldn't want to call it "Smurf"